### PR TITLE
[docs] [#11029]  Add "Contribute" section and integrate bugyard

### PR DESCRIPTION
**Pull request by rajmaddy89**
Description
--
Please refer to this issue: https://github.com/yugabyte/yugabyte-db/issues/11029

This PR adds a dropdown "Contribute" functionality for readers of the doc page to enable them the following actions:
1. `Report a doc issue` should take you to a GitHub issue template and include the URL of the page you were on.
2. `Suggest new content` should take you to a different GitHub issue template, and include the URL of the page you were on.

Also I have marked the PR with `DO NOT MERGE` tag as I have also done the bugyard integration for the people to play with. Before merging, we need to ensure to remove that integration piece.

TODO: For `Suggest new content`, we need to add one more label and @polarweasel will clarify on that

### CODE REVIEW

File: docs/layouts/partials/contribute_list.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="dropdown" id="contribute-list">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownContributeButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><strong>
+        Contribute</strong>
+    </button>
+
+    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+{{ $.Page.Title }}&labels=area/documentation&body={{ .Permalink }}%0A%0A%23%23%20Issue%20Description%0A%0A%23%23%20Suggested%20Resolution">
+            <strong>Report Doc Issue</strong>
+        </a>
+        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+New+Docs+Proposal&labels=area/documentation,community/request&body={{ .Permalink }}%0A%0A%23%23%20Issue%20Description%0A%0A%23%23%20Suggested%20Resolution">
```
Comment (polarweasel) [inline docs/layouts/partials/contribute_list.html:10]: https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+New+docs+proposal&labels=area/documentation,community/request&body=https%3a%2f%2fdocs.yugabyte.com%2flatest%2fexplore%2ffault-tolerance%2fmacos%2f%0A%0A%23%23%20Issue%20description%0A%0A%23%23%20Suggested%20improvement

```suggestion
        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+New+docs+proposal&labels=area/documentation,community/request&body={{ .Permalink }}%0A%0A%23%23%20Issue%20description%0A%0A%23%23%20Suggested%20improvement">
```

File: docs/layouts/partials/contribute_list.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+            <strong>Suggest New Content</strong>
```
Comment (polarweasel) [inline docs/layouts/partials/contribute_list.html:11]:
```suggestion
            <strong>Suggest new content</strong>
```

File: docs/layouts/partials/contribute_list.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="dropdown" id="contribute-list">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownContributeButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><strong>
+        Contribute</strong>
+    </button>
+
+    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+{{ $.Page.Title }}&labels=area/documentation&body={{ .Permalink }}%0A%0A%23%23%20Issue%20Description%0A%0A%23%23%20Suggested%20Resolution">
+            <strong>Report Doc Issue</strong>
```
Comment (polarweasel) [inline docs/layouts/partials/contribute_list.html:8]:
```suggestion
            <strong>Report a doc issue</strong>
```

File: docs/layouts/partials/head.html [hunk 4]
```diff
@@ -6,6 +6,9 @@
   <!--[if (gt IE 9)|!(IE)]><!--> <html class="no-js" {{ with .Site.LanguageCode }}lang="{{ . }}"{{ end }}> <!--<![endif]-->
 
   <head>
+    <script type="text/javascript">
+      !function(){if("function"!=typeof window.bugyard){var a=function(){a.c(arguments)};a.q=[],a.c=function(b){a.q.push(b)},window.bugyard=a;var b=document.createElement("script");b.setAttribute("data-bugyard","61e0ec1038569600142ffcd2"),b.setAttribute("async","async"),b.setAttribute("defer","defer"),b.setAttribute("src","https://widget.bugyard.io/bugyard.min.js"),document.getElementsByTagName("head")[0].appendChild(b)}}();
+    </script>
```
Comment (polarweasel) [inline docs/layouts/partials/head.html:11]: All other scripts are down around #L47 -- should this be there, as well, or does it need to be up top here?

Also, let's comment this one out until we have Bugyard approved for use in production. :)

```suggestion
    <!--
    <script type="text/javascript">
      !function(){if("function"!=typeof window.bugyard){var a=function(){a.c(arguments)};a.q=[],a.c=function(b){a.q.push(b)},window.bugyard=a;var b=document.createElement("script");b.setAttribute("data-bugyard","61e0ec1038569600142ffcd2"),b.setAttribute("async","async"),b.setAttribute("defer","defer"),b.setAttribute("src","https://widget.bugyard.io/bugyard.min.js"),document.getElementsByTagName("head")[0].appendChild(b)}}();
    </script>
    -->
```

File: docs/styles/_contribute_list.scss [hunk 5]
```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Yugabyte, Inc.
+
+@import "./_colors.scss";
+
+#contribute-list{
```
Comment (polarweasel) [inline docs/styles/_contribute_list.scss:5]: As we just talked about, I think the dropdown needs a little more contrast to help it get noticed. Nothing outrageous, but a bit more noticeable would be good.

**Review by polarweasel — CHANGES_REQUESTED** [review]
Looking good! Left a couple of suggestions.

File: docs/styles/_contribute_list.scss (hunk 5 above)
Comment (rajmaddy89) [reply docs/styles/_contribute_list.scss:5]: I have set the opacity to `0.2` for the dropdown, if you think its bright, i can change it to `0.1`

Comment (polarweasel) [reply docs/styles/_contribute_list.scss:5]: Let's go with 0.2 for now. 0.1 feels too light, and 0.15 feels too nitpicky :)

**Review by polarweasel — APPROVED** [review]

**Pull request merged**